python/src/DistFunc_doc.i.in

```diff
@@ -464,32 +552,36 @@ Examples
 // ---------------------------------------------------------------------
 
 %feature("docstring") OT::DistFunc::pPearsonCorrelation
-"Asymptotic probability function for the Pearson :math:`\rho` correlation.
+"Cumulative density function  of the Pearson statistics distribution.
 
 Parameters
 ----------
 n : int
-    The size of the population
+    The size of the samples.
 
 rho : float :math:`-1<rho<1`
-    The Pearson correlation coefficient
+    The location.
```

Comment:
You mean the location parameter of the distribution of the Pearson sample correlation coefficient? It seems to me that the original naming was clearer.

Reply:
Yes....